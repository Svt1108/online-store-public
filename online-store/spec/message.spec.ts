import MessageWindow from "../src/components/message";

describe("Method createMessage", () => {
  const testMessageWindow = new MessageWindow();

  test("should be defined", () => {
    expect(testMessageWindow.createMessage).toBeDefined;
  });

  it("create correct message", () => {
    const text = "This is a message";
    testMessageWindow.createMessage(text);
    expect(document.body.innerHTML).toMatchSnapshot();
  });
});
