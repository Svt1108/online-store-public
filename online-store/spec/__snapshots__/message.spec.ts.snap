// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Method createMessage create correct message 1`] = `"<div class=\\"modal-wrap\\"><div class=\\"message-window\\"><p class=\\"message-window__text\\">This is a message</p></div></div>"`;
